test-app/startup_arm.c

```diff
@@ -283,7 +286,7 @@ void (* const IV[])(void) =
     isr_empty, //	UCPD1_IRQHandler
     isr_empty, //	ICACHE_IRQHandler
     isr_empty, //	OTFDEC1_IRQHandler
-#else /* For STM32F4 */
+#elif STM32 /* For STM32F4 */
```

Comment:
Also if someone is building outside our make system you should also check `defined(STM32)`.